app/src/main/java/com/hannesdorfmann/adapterdelegates3/sample/adapterdelegates/DiffDogAdapterDelegate.java

```diff
@@ -0,0 +1,62 @@
+package com.hannesdorfmann.adapterdelegates3.sample.adapterdelegates;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.hannesdorfmann.adapterdelegates.sample.R;
+import com.hannesdorfmann.adapterdelegates3.AbsListItemAdapterDelegate;
+import com.hannesdorfmann.adapterdelegates3.DiffItem;
+import com.hannesdorfmann.adapterdelegates3.sample.model.DiffDog;
+
+import java.util.List;
+
+/**
+ * Created by six_hundreds on 15.06.18.
+ */
```

Comment:
Please remove this comment or make it more meaningful (I think removing it is better).